creator-node/nginx/creator_nginx.conf

```diff
@@ -0,0 +1,156 @@
+# nginx Configuration file
```

Comment:
I like the idea of baking this config into the docker image itself so there are fewer options for service operators to footgun themselves, could we do that instead of just depending on the public openresty?